src/dmd/cli.d

```diff
@@ -545,6 +545,14 @@ dmd -cov -unittest myprog.d
             `$(UNIX Generate shared library)
              $(WINDOWS Generate DLL library)`,
         ),
+        Option("std=<standard>",
+            "set compatiblity with $(I standard)",
```

Comment:
First line isn't DDoc I think, it's what ends up in `-help`. This is what appears to be failing the doc build.

Looking for a precedent, it doesn't look like any special formatting is used for arguments in command switch help text.

Reply:
fixed

Reply:
I think the second line might not be either, judging by the rest of the code.

Reply:
See my reply above.

Reply:
> fixed

I don't think it went through:

> 0b98fd989cb73e5cfe3af04767fc32f59c7a1e61 and a5ebdedff7fae08f53bdabdc8297c4c933fefdf2 are identical.